README.md

```diff
@@ -124,6 +125,11 @@ Technical writing is writing or drafting technical communication used in technic
 * [The Red Hat Style Guide](https://stylepedia.net/style/)
 * [Google Developer Documentation Style Guide](https://developers.google.com/style/)
 
+## Blogs
```

Comment:
I don't think we would need a "Blog" section as there are tons of blogs we would want to recommend instead we should use the "Technical Writers to Follow Section" which contains the name, niche, and link to blogs of notable writers you might want to checkout.

Or can you explain why you want this section?